plugins/tileacq.py

```diff
@@ -288,11 +314,27 @@ def show_dlg(self):
         ss = self._get_live_streams()
         self.filename.value = self._get_new_filename()
 
+        # On SECOM allow to do fine alignment for each tile
+        if self._can_fine_align(ss):
+            self.vaconf["fine_align"] = {"label": "Fine alignment"}
+            # Set to True to make it the default, but will be automatically
+            # disabled later if the current visible streams don't allow it.
+            self.fine_align.value = True
+            for s in ss:
```

Comment:
Let's move this to a separate function `create_overlay_stream()`. It should be clearer.